frontend/src/components/AdminPanel/OrderDetailsCard/index.tsx

```diff
@@ -0,0 +1,253 @@
+import { Typography, styled, Box, useTheme } from '@mui/material';
+import  Clear  from "@mui/icons-material/clear"
```

Comment:
@rohansen856  double check this  line @mui/icons-material/clear"

Reply:
I will fix the issue with the formatting and add the semicolon(`;`) in this line.

Reply:
@rohansen856 nah the probelm is the its "@mui/icons-material/Clear" , on my end i see it giving a error for ...../clear